src/js/components/AppListComponent.jsx

```diff
@@ -178,7 +178,7 @@ var AppListComponent = React.createClass({
     var filterLabels = filters[FilterTypes.LABELS];
     var filterStatus = filters[FilterTypes.STATUS];
 
-    if (!Util.isEmptyString(filterText)) {
+    if (filterText != null && filterText !== "") {
```

Comment:
Yeah, this name is much better!
This function has a specific and meaningful use-case in our form transformator and validation functions.
But we really should rename it.